examples/Tables/Table.md

```diff
@@ -232,33 +234,60 @@ import {
   <TableBody>
     <TableRow>
       <TableRowHeadingCell scope="row">CS 50</TableRowHeadingCell>
-      <TableCell>YES</TableCell>
+      <TableCell>
+        <TableCellList>
+          <TableCellListItem>
+            Glenn, Kristin
+          </TableCellListItem>
+          <TableCellListItem>
+            Thompson, Jack
+          </TableCellListItem>
+        </TableCellList>
+      </TableCell>
       <TableCell>600</TableCell>
       <TableCell>480</TableCell>
       <TableCell>300</TableCell>
-      <TableCell>NO</TableCell>
+      <TableCell></TableCell>
       <TableCell />
       <TableCell />
       <TableCell />
     </TableRow>
     <TableRow isStriped>
       <TableRowHeadingCell scope="row">CS 51</TableRowHeadingCell>
-      <TableCell>NO</TableCell>
+      <TableCell></TableCell>
       <TableCell />
       <TableCell />
       <TableCell />
-      <TableCell>YES</TableCell>
+      <TableCell>
+        <TableCellList>
+          <TableCellListItem>
+            Michaels, Lianne
+          </TableCellListItem>
+        </TableCellList>
```

Comment:
Just a question for my own understanding - Is the purpose of having a `<TableCellList>` with only one `<TableCellListItem>` that these are cells that would potentially have multiple list items in the future?

Reply:
I don't know if it's necessarily about "the future" so much as it is about consistency within the column, ie., showing that instructors _can_ have multiple entries, even if this specific cell only has one. If we're thinking in **types**, it's basically the difference between `Array<string>` and `string`.

I should also note that in practical usage, this would most likely be done by iterating through an array:

```tsx
<TableCellList>
  {instructors.map(
    ({ id, displayName }): ReactElement => (
      <TableCellListItem key={id}>
          {displayName}
      </TableCellListItem>
    )
  )}
</TableCellList>

```

Reply:
Ok that makes sense